docs/installation/advanced/seadoc.md

```diff
@@ -0,0 +1,126 @@
+# SeaDoc Integration
+
+SeaDoc is an extension of SeaTable that providing an online collaborative document editor.
+
+SeaDoc designed around the following key ideas:
+
+* An expressive easy to use editor
+* A review and approval workflow to better control how contents changes
+* Inter-document linking for connecting related contents
+* AI integration that streamlines content generation, summarization, and management
+* Comprehensive APIs for automating document generating and processing
+
+SeaDoc excels at:
+
+* Authoring product and technical documents
+* Creating knowledge base articles and online manuals
+* Building internal Wikis
+
+## Architecture
+
+The SeaDoc archticture is demonstrated as below:
+
+![SeaDoc](../../assets/images/seadoc-arch.png)
+
+Here is the workflow when a user open a seadoc in browser
+
+1. When a user open a seadoc in the browser, a file loading request will be sent to Caddy, and Caddy proxy the request to SeaDoc server.
+2. SeaDoc server will send the file's content back if it is already cached, otherwise SeaDoc serve will sends a request to SeaTable server.
+3. SeaTable server loads the content, then sends it to SeaDoc server and write it to the cache at the same time.
+4. After SeaDoc receives the content, it will be sent to the browser.
+
+## Deployment SeaDoc
+
+The easiest way to deployment SeaDoc is to deploy it with SeaTable server on the same host using the same Docker network. If in some situations, you need to deployment SeaDoc standalone, you can follow the next section.
+
+### Change the .env file
+
+To install SeaDoc, include `seadoc.yml` in the `COMPOSE_FILE` variable within your `.env` file. This instructs Docker to download the required images for SeaDoc.
+
+Simply copy and paste (:material-content-copy:) the following code into your command line:
+
+```bash
+sed -i "s/COMPOSE_FILE='\(.*\)'/COMPOSE_FILE='\1,seadoc.yml'/" /opt/seatable-compose/.env
+```
+
+### Add SeaDoc configurations to .env file
+
+```env
+DTABLE_PRIVATE_KEY=
+ENABLE_SEADOC=true
+SEADOC_SERVER_URL='http://seatable.example.com:7070'
+```
+
+Note: SEADOC used port 7070.
+
+`DTABLE_PRIVATE_KEY` copy form `/opt/seatable/conf/dtable_web_settings.py` in SeaTable container.
+
+### Download SeaDoc and restart
+
+One more step is necessary to download the SeaDoc image and restart the SeaTable service.
+
+```bash
+cd /opt/seatable-compose
+docker compose down
+docker compose up -d
```

Comment:
`docker compose up -d` is enough and causes less containers to restart. docker-compose automatically detects which containers need to be restarted